soc/nxp/imxrt/imxrt11xx/soc.c

```diff
@@ -55,22 +55,30 @@
 #endif
 
 #ifdef CONFIG_INIT_ARM_PLL
-static const clock_arm_pll_config_t armPllConfig = {
-#if defined(CONFIG_SOC_MIMXRT1176_CM4) || defined(CONFIG_SOC_MIMXRT1176_CM7)
-	/* resulting frequency: 24 * (166/(2* 2)) = 984MHz */
-	/* Post divider, 0 - DIV by 2, 1 - DIV by 4, 2 - DIV by 8, 3 - DIV by 1 */
-	.postDivider = kCLOCK_PllPostDiv2,
-	/* PLL Loop divider, Fout = Fin * ( loopDivider / ( 2 * postDivider ) ) */
-	.loopDivider = 166,
-#elif defined(CONFIG_SOC_MIMXRT1166_CM4) || defined(CONFIG_SOC_MIMXRT1166_CM7)
-	/* resulting frequency: 24 * (200/(2 * 4)) = 600MHz */
-	/* Post divider, 0 - DIV by 2, 1 - DIV by 4, 2 - DIV by 8, 3 - DIV by 1 */
-	.postDivider = kCLOCK_PllPostDiv4,
-	/* PLL Loop divider, Fout = Fin * ( loopDivider / ( 2 * postDivider ) ) */
-	.loopDivider = 200,
+
+#if defined(CONFIG_SOC_MIMXRT1176)
+#define DEFAULT_LOOPDIV 83
+#define DEFAULT_POSTDIV 2
```

Comment:
Can we get these values from DTS?
```
&arm_pll {
	clock-mult = <83>;
	clock-div = <2>;
}
```

Reply:
Yes, we are doing that in this PR. This was added because of @decsny's suggestion that we retain default values here, so users with downstream boards do not need to modify their board port after this PR merges

Reply:
would have been nicer to do DT_PROP_OR with another macro with COND_CODE of the soc kconfig